scheduler/scheduler-server/src/test/java/org/ow2/proactive/scheduler/core/JobEmailNotificationTest.java

```diff
@@ -302,6 +302,26 @@ public void testFinished() throws Exception {
         verifyNoMoreInteractions(stubbedSender);
     }
 
+    @Test
+    public void testFinishedWithErrors() throws Exception {
```

Comment:
you should add a test to verify that, if a job contains no error, with NOTIFICATION_EVENTS sets the same as this test, no notification should occur.